pylint_odoo/test_repo/broken_module/__init__.py

```diff
@@ -2,3 +2,4 @@
 
 from . import models
 from .models import broken_model
+from . import tests
```

Comment:
You could test also the form:

`from tests import something`

Reply:
Done

Reply:
did you mean?

`from .tests import something`

Notice the dot at start

It is not the same.

Reply:
Done